db/controllers/articles.controller.js

```diff
@@ -15,4 +20,30 @@ exports.getArticleByID = (req, res, next) => {
       res.status(200).send({ article });
     })
     .catch(next);
-};
\ No newline at end of file
+};
+
+// exports.updateArticleByID = (req, res, next) => {
+//   const updatedVotes = req.body.inc_votes;
+//   const { article_id } = req.params;
+
+//   patchArticleByID(updatedVotes, article_id)
+//     .then((updatedArticle) => {
+//       res.status(200).send({ updatedArticle });
+//     })
+//     .catch(next);
+// };
+
+exports.updateArticleByID = (req, res, next) => {
+  const updatedVotes = req.body.inc_votes;
+  const { article_id } = req.params;
+
+  const promises = [
+    checkArticleExists(article_id), patchArticleByID(updatedVotes, article_id)
```

Comment:
This works fine, but you don't actually need to use the checkArticleExists function for this task. You can just throw a Promise.reject from your model, which can then get picked up by your error handling middleware 👍